app/feedme/tasks.py

```
@@ -2201,13 +2228,38 @@ def _redact_log_findings(
                     ticket_id,
-                    str(exc)[:180],
+                    error_text[:180],
                 )
-                failed += 1
-                continue
+                try:
+                    model = service._get_embedding_model()
+                    loop = asyncio.get_running_loop()
+                    embedding = await loop.run_in_executor(None, model.embed_query, content)
+                    assert_dim(embedding, context="zendesk_import_fallback")
+                except Exception as fallback_exc:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)